.dockerignore

```diff
@@ -0,0 +1,15 @@
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+*.db
+*.env
+README.md
+Dockerfile
+.venv
+result.png
+checklist.md
+.gitignore
+.github
+.idea
```

Comment:
The .idea directory is specific to IDE settings and should not be included in the Docker image, which is correctly ignored here.